packages/components/src/presentation-components/properties/inputs/UseQuantityValueInput.tsx

```diff
@@ -75,7 +75,7 @@ export function useQuantityValueInput({ initialRawValue, schemaContext, koqName
         placeholder: newPlaceholder,
       };
     });
-  }, [formatter, parser]);
+  }, [formatter, initialRawValue, parser]);
```

Comment:
I think we should store `initialRawValue` in `useRef` for this to avoid rerenders if different initial value is provided.

Reply:
Changed